sw/device/lib/dif/dif_uart.h

```diff
@@ -94,22 +94,22 @@ typedef enum dif_uart_watermark {
 } dif_uart_watermark_t;
 
 /**
- * A UART FIFO reset selection.
+ * A UART datapath to select various enables / FIFO resets.
  */
-typedef enum dif_uart_fifo_reset {
+typedef enum dif_uart_datapath {
   /**
-   * Indicates that the RX FIFO should be reset.
+   * Selects the RX datapath for enablement / reset.
    */
-  kDifUartFifoResetRx = 0,
+  kDifUartDatapathRx = 0,
   /**
-   * Indicates that the TX FIFO should be reset.
+   * Selects the TX datapath for enablement / reset.
    */
-  kDifUartFifoResetTx,
+  kDifUartDatapathTx,
   /**
-   * Indicates that both FIFOs should be reset.
+   * Selects both the RX and TX datapaths for enablement / reset.
    */
-  kDifUartFifoResetAll,
-} dif_uart_fifo_reset_t;
+  kDifUartDatapathAll,
```

Comment:
minor nit: Consider using bitmasks when defining enums with set semantics in the future:

```
kRx = 0x1,
kTx = 0x2,
kRxTx = 0x3,
```

Example Usage:
```
if (enable & kRx) {
  // Enable RX
}
If (enable & kTx) {
  // Enable Tx
}
```